File: fabric/src/main/java/dev/aika/bigclever/fabric/BigCleverFabric.java
package dev.aika.bigclever.fabric;

import net.fabricmc.api.ModInitializer;

import dev.aika.bigclever.BigClever;

public final class BigCleverFabric implements ModInitializer {
    @Override
    public void onInitialize() {
        BigClever.init();
    }
}

File: fabric/src/main/kotlin/dev/aika/bigclever/fabric/BigCleverFabric.kt
package dev.aika.bigclever.fabric

import dev.aika.bigclever.BigClever
import net.fabricmc.api.ModInitializer

@Suppress("unused")
class BigCleverFabric : ModInitializer {
    override fun onInitialize() {
        BigClever.init()
    }
}